packages/gui/components/ModalManager/ModalContainer.tsx

```diff
@@ -73,6 +75,11 @@ export const ModalContainer: FC<Props> = ({ item }) => {
 			opacity: opacity.value,
 		};
 		if (fullWidth) baseStyle.width = width.value;
+		if (fullHeight) {
+			baseStyle.height = height.value;
+			baseStyle.top = 0;
+			baseStyle.left = 0;
```

Comment:
we should remove `left` because it does not relate to height binding. Also, we currently treat modal as bottom sheet, so binding the bottom to 0 is more suitable than the top.